libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/CacheResultUtils.kt

```diff
@@ -0,0 +1,43 @@
+package com.mapbox.navigation.ui.maps.internal.route.line
```

Comment:
```suggestion
package com.mapbox.navigation.ui.maps.route.line
```

NIT: It doesn't have to be in the `internal` package if the class is actually `internal`. `internal` package should contain classes that are `public`, but shouldn't be used.